test/over_react_redux/connect_flux_test.dart

```diff
@@ -482,18 +497,28 @@ main() {
               areStatesEqual: (_, __) => false,
             )(Counter);
 
+            // In this setup with an idiomatic redux store, w'd expect double the updates when the state is updated
```

Comment:
```suggestion
            // In this setup with an idiomatic redux store, we'd expect double the updates when the state is updated
```

Reply:
Addressed in 93fb99c173be4f1da9f8630d34680cda808774b5